### DISCUSSION TITLE
New release, with support to new frontend

### DISCUSSION THREAD

**Discussion by crystian** [crystian/ComfyUI-Crystools#132] [Announcements]
That supports the old, current, and new versions of frontend clients!

**Comment by adamreading** [comment]
Hi - Progress Bar is no longer visible in latest version of Beta UI? Was working fine before ComfyUI updated.

**Comment by crystian** [comment]
umm, works, with 1.3.11 (latest) and 1.3.22 (beta)
may you need to clean your data (try with incognito please)
